launch/display_rover.launch

```diff
@@ -1,13 +1,14 @@
 <launch>
   <arg name="model" default="$(find mrover)/rover_description/rover.urdf"/>
   <arg name="gui" default="false" />
-  <param name="robot_description" command="cat $(arg model)" />
+  <param name="robot_description" command="xacro $(find mrover)/rover_description/rover.xacro"/>
   <arg name="rvizconfig" default="$(find mrover)/default.rviz" />
 
   <node if="$(arg gui)" name="joint_state_publisher" pkg="joint_state_publisher_gui" type="joint_state_publisher_gui" />
   <node unless="$(arg gui)" name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" />
   <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" />
   <node name="rviz" pkg="rviz" type="rviz" args="-d $(arg rvizconfig)" required="true" />
+  <node pkg="tf" type="static_transform_publisher" name="base_to_camera" args=" 0 0 0.2 -1.570796327 0 -1.570796327 base_link camera_link 100" />
```

Comment:
lets just add a rigid camera link to the URDF instead of publishing a static transform like this, you can access baes -> camera In the TF tree anyway

Reply:
See comment below